pygeoapi/process/manager/tinydb_.py

```diff
@@ -27,7 +27,12 @@
 #
 # =================================================================
 
-import fcntl
+try:
+    import fcntl
+except ModuleNotFoundError:
+    # When on Windows, fcntl does not exist and file locking is automatic
```

Comment:
There really was no need to include this fix in this PR, but it made my life a lot easier (yes, I'm on Windows...), because the `ModuleNotFoundError` was raised on every API test.
For quicker testing, I do not wish to spin up a Linux Docker container all the time...